### ISSUE TITLE
Remove internal suspense boundary

### ISSUE THREAD

**Issue by alexandernanberg** [pmndrs/use-cannon#298]
I can't see a good reason why there is a suspense boundary inside the library, this should ideally be handled by consumers for more granular control. E.g. consumer might not want to render `null`.

https://github.com/pmndrs/use-cannon/blob/543f3e243947ff68488f549208417289cde60fdd/src/index.tsx#L8-L14

This would unfortunately be a breaking change, but in the long run I think it's a good one.

**Comment by bjornstar** [comment]
Would it be useful to provide a fallback property to the props?

**Comment by alexandernanberg** [comment]
Not really for my use-case tbh, because it doesn't enable compostability in the same way. 

E.g. imagine this scenario 
```jsx
<Canvas>
	<Suspense fallback="Loading...">
		<ComponentThatLoadsAsset />
		<Physics>
			<ComponentThatLoadsAsset />
		</Physics>
	</Suspense>
</Canvas>
```

**Comment by drcmda** [comment]
i agree, it would be more flexible if it's in userland. it is a breaking change though, would require a new major.

**Comment by alexandernanberg** [comment]
@drcmda Oh right, good point, was just quickly typing out pseudo-code 😄
